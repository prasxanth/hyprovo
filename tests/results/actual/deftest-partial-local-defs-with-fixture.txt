2021-11-24 17:25:52.077 - INFO - Pass ✓... (deftest-partial-local-defs-with-fixture / test-sub-variable-partial-local-defs): add pass
2021-11-24 17:25:52.079 - ERROR - Fail ✗... (deftest-partial-local-defs-with-fixture / test-sub-variable-partial-local-defs): add fail
2021-11-24 17:25:52.084 - INFO - Pass ✓... (deftest-partial-local-defs-with-fixture / test-mul-variable-partial-local-defs): mul pass
2021-11-24 17:25:52.086 - ERROR - Fail ✗... (deftest-partial-local-defs-with-fixture / test-mul-variable-partial-local-defs): mul fail
2021-11-24 17:25:52.086 - DEBUG - (deftest-partial-local-defs-with-fixture): Test Summary → Total: 4    Pass: 2    Fail: 2
