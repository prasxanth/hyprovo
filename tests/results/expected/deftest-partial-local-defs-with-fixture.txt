2021-11-24 17:24:55.650 - INFO - Pass ✓... (deftest-partial-local-defs-with-fixture / test-sub-variable-partial-local-defs): add pass
2021-11-24 17:24:55.652 - ERROR - Fail ✗... (deftest-partial-local-defs-with-fixture / test-sub-variable-partial-local-defs): add fail
2021-11-24 17:24:55.658 - INFO - Pass ✓... (deftest-partial-local-defs-with-fixture / test-mul-variable-partial-local-defs): mul pass
2021-11-24 17:24:55.660 - ERROR - Fail ✗... (deftest-partial-local-defs-with-fixture / test-mul-variable-partial-local-defs): mul fail
2021-11-24 17:24:55.660 - DEBUG - (deftest-partial-local-defs-with-fixture): Test Summary → Total: 4    Pass: 2    Fail: 2
